### DISCUSSION TITLE
How to get Yolo badge ?

### DISCUSSION THREAD

**Discussion by sanrionpc** [community/community#19981] [Other Feature Feedback, Questions, & Ideas] [answered]
How to get yolo badge

**Comment by DxstyLy** [comment, marked as answer]
To get YOLO badge, you need to merge a Pull Request directly into a public code repo without review.

Hope this clarifies.

**Comment by GreatHorse115** [comment]
Thank you.

**Comment by bethwel3001** [comment]
To get YOLO badge, you need to merge a Pull Request directly into a public code repo without a review.

Hope this helps.
